integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -210,28 +225,24 @@ def get_filtered_groups_for_webhooks(
 
         return groups_for_webhooks
 
-    def create_webhooks(self, groups_for_webhooks) -> list[int | str]:
-        # Filter out groups that are not in the group mapping and creating webhooks for the rest
-        logger.info(
-            f"Creating webhooks for the groups: {[group.attributes['full_path'] for group in groups_for_webhooks]}"
-        )
-        webhook_ids = []
-        for partial_group in groups_for_webhooks:
-            group_id = partial_group.get_id()
-            if group_id is None:
-                logger.info(
-                    f"Group {partial_group.attributes['full_path']} has no id. skipping..."
-                )
+    def create_webhook(
+        self, group: Group, events: list[str] | None
+    ) -> int | str | None:
```

Comment:
how come it can return both int and str?

Reply:
leftover mistake, fixed